### PR TITLE
Issue 70

### PR DESCRIPTION
**Pull request by JEAlfonsoP**
require CC0 waiver.

### CODE REVIEW

File: src/modules/assertions/require_cc0.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  GITHUB_TOKEN,
+  GithubSelf,
+  PR,
+  Review
+} from "src/domain";
+import _ from "lodash";
+
+export const  requireCC0 = () => {
+  const github = getOctokit(GITHUB_TOKEN).rest;
+  const copyrightRegex = /^[\s\S]*\n## Copyright\nCopyright and related rights waived via \[CC0\]\(\.\.\/LICENSE.md\)\.\n?$/g;
```
Comment (MicahZoltu) [inline src/modules/assertions/require_cc0.ts:13]:
```suggestion
  const copyrightRegex = /^[\s\S]*\n## Copyright\n{1,2}Copyright and related rights waived via \[CC0\]\(\.\.\/LICENSE.md\)\.\n?$/g;
```
Some people put a blank line below headings.  Would be nice if we could support that style.

**Comment by MicahZoltu** [comment]
CI is failing.  Please investigate and address then mention me when it is passing.

**Comment by JEAlfonsoP** [comment]
@MicahZoltu done

**Comment by MicahZoltu** [comment]
Looks like this branch has conflicts now that the other was merged.  You *should* be able to just click the Resolve Conflicts button in GitHub to address the issue.

**Comment by MicahZoltu** [comment]
Oh, this changeset appears to include the other PR as well.  It should only include the changes relevant to this PR.

**Comment by JEAlfonsoP** [comment]
Sorry, It looks like they all took the requirePullNumber() remove line in /src/main.ts
I did resolve the merge conflict. If you want a cleaner PRs I will delete these and create new ones.

**Comment by JEAlfonsoP** [comment]
Fixes #70

**Review by MicahZoltu — CHANGES_REQUESTED** [review]
Sorry, just realized I never clicked "submit review" on this.  😖

File: src/modules/assertions/require_cc0.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { context } from "@actions/github";
+import _ from "lodash";
+
+export const  requireCC0 = () => {
+  const copyrightRegex = /^[\s\S]*\n## Copyright\n{1,2}Copyright and related rights waived via \[CC0\]\(\.\.\/LICENSE.md\)\.\n?$/g;
+  if (context.payload?.pull_request?.body && !(copyrightRegex.test(context.payload?.pull_request?.body))) {
```
Comment (MicahZoltu) [inline src/modules/assertions/require_cc0.ts:6]: TypeScript should be smart enough to figure out that we have already handled the falsy cases, so we don't need the null coalescing operator after the `&&`.
```suggestion
  if (context.payload?.pull_request?.body && !(copyrightRegex.test(context.payload.pull_request.body))) {
```

**Comment by MicahZoltu** [comment]
This is now handled by EIPW.  See my comment on #89 discussing how we should decide whether something goes into this bot or EIPW.

**Comment by JEAlfonsoP** [comment]
will you reopen this PR ?

**Pull request closed without merging**